src/commands/synthetics/run-test.ts

```diff
@@ -220,11 +253,18 @@ export class RunTestCommand extends Command {
   }
 }
 
+const removeUndefinedValues = <T extends {[key: string]: any}>(object: T): T => {
+  const newObject = {...object}
+  Object.keys(newObject).forEach((k) => newObject[k] === undefined && delete newObject[k])
+
+  return newObject
+}
+
 RunTestCommand.addPath('synthetics', 'run-tests')
 RunTestCommand.addOption('apiKey', Command.String('--apiKey'))
 RunTestCommand.addOption('appKey', Command.String('--appKey'))
 RunTestCommand.addOption('configPath', Command.String('--config'))
+RunTestCommand.addOption('fileGlobs', Command.Array('-f,--files'))
```

Comment:
Should be renamed as `files` to unify the naming with the internal configuration.

Reply:
N.B: I merged `this.files` and `this.fileGlobs` to `this.fileGlobs` because `fileGlobs` was already used as a CLI option and `file` was only use internally for a  default value.

Reply:
But doesn't it change the config file API?
So we also need to change all the docs we have around it, and also be sure we bump a minor on this.

IMO, not worth the pain.